Journal/Journal/View Controller/EntriesTableViewController.swift

```diff
@@ -0,0 +1,157 @@
+//
+//  EntriesTableViewController.swift
+//  Journal
+//
+//  Created by Norlan Tibanear on 8/9/20.
+//  Copyright © 2020 Norlan Tibanear. All rights reserved.
+//
+
+import UIKit
+import CoreData
+
+class EntriesTableViewController: UITableViewController {
+    
+    
+    // MARK: - Properties
+    
+    lazy var fetchedResultsController: NSFetchedResultsController<Entry> = {
+        let fetchRequest: NSFetchRequest<Entry> = Entry.fetchRequest()
+        fetchRequest.sortDescriptors = [
+            NSSortDescriptor(key: "title", ascending: true),
+            NSSortDescriptor(key: "timestamp", ascending: true)
+            ]
```

Comment:
It's a little thing but make sure things are indented properly, with your closing brace lined up with the starting line:

``` swift
fetchRequest.sortDescriptors = [
    NSSortDescriptor(key: "title", ascending: true),
    NSSortDescriptor(key: "timestamp", ascending: true)
]
```